src/SongManager.cpp

```diff
@@ -521,6 +521,15 @@ void SongManager::LoadSongDir( RString sDir, LoadingWindow *ld, bool onlyAdditio
 	}
 }
 
+Group* SongManager::GetGroupByName(const RString& groupName) const {
+	for (const auto& group : m_vGroups) {
+		if (group.GetGroupName() == groupName) {

```

Comment:
There is already a function for this ``GetGroupFromName``. I'd say to use the existing function or suggest changing it for there's something that should be changed.